addons/point_of_sale/static/src/js/models.js

```diff
@@ -2722,17 +2722,16 @@ class Order extends PosModel {
             return sum + orderLine.get_price_without_tax();
         }), 0), this.pos.currency.rounding);
     }
+    _reduce_total_discount_callback(sum, orderLine) {
+        sum += (orderLine.get_unit_price() * (orderLine.get_discount()/100) * orderLine.get_quantity());
+        if (orderLine.display_discount_policy() === 'without_discount'){
+            sum += ((orderLine.get_lst_price() - orderLine.get_unit_price()) * orderLine.get_quantity());
+        }
+        return sum;
+    }
     get_total_discount() {
-        return round_pr(this.orderlines.reduce((function(sum, orderLine) {
-            if (orderLine.pos.config.down_payment_product_id[0] === orderLine.product.id) {
-                return 0;
-            }
-            sum += (orderLine.get_unit_price() * (orderLine.get_discount()/100) * orderLine.get_quantity());
-            if (orderLine.display_discount_policy() === 'without_discount'){
-                sum += ((orderLine.get_lst_price() - orderLine.get_unit_price()) * orderLine.get_quantity());
-            }
-            return sum;
-        }), 0), this.pos.currency.rounding);
+        const reduce_callback = this._reduce_total_discount_callback.bind(this);
+        return round_pr(this.orderlines.reduce(reduce_callback, 0), this.pos.currency.rounding);
```

Comment:
You can do this so that you don't need to manually bind.
```suggestion
        return round_pr(this.orderlines.reduce((t, l) => this._reduce_total_discount_callback(t, l), 0), this.pos.currency.rounding);
```